saleor/graphql/account/types.py

```diff
@@ -120,8 +120,10 @@ def resolve_is_default_billing_address(root: models.Address, _info):
         return False
 
     @staticmethod
-    def __resolve_reference(root, _info, **_kwargs):
-        return graphene.Node.get_node_from_global_id(_info, root.id)
+    def __resolve_reference(root: "Address", info, **_kwargs):
+        from .resolvers import resolve_address
+
+        return resolve_address(info, root.id, raise_exception=False)
```

Comment:
Maybe we can catch an error here instead of adding new param to the resolver? 
@maarcingebala what do you think?

Reply:
Yep, I think this would be a more elegant solution.

Reply:
I've actually did that and changed it other way because function itself didn't reflect that usage scenario 😅

But fine, this is now covered with tests so any breaking change to the function will be easy to catch here anyway.